packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-ciemss-drilldown.vue

```diff
@@ -5,137 +5,178 @@
+							</div>
+
+							<div class="label-and-input">
+								<label> Preset </label>
+								<Dropdown
+									class="flex-1"
+									v-model="knobs.extra.presetType"
+									placeholder="Select an option"
+									:options="[CiemssPresetTypes.Fast, CiemssPresetTypes.Normal]"
+									@update:model-value="setPresetValues"
+								/>
+								<label class="mb-1 p-text-secondary text-sm">
+									<i class="pi pi-info-circle" />
+									This impacts solver method, iterations and learning rate.
+								</label>
+							</div>
+
+							<fieldset class="mt-1 additional-settings">
+								<div class="label-and-input">
+									<label>Number of Samples</label>
+									<tera-input-number v-model="knobs.extra.numParticles" @update:model-value="updateState" />
+								</div>
+								<div class="spacer m-3" />
+
+								<h6 class="mb-2">ODE solver options</h6>
+
+								<div class="input-row">
+									<div class="label-and-input">
+										<label>Solver method</label>
 										<Dropdown
-											class="p-inputtext-sm"
-											:options="['dopri5', 'euler']"
 											v-model="knobs.extra.solverMethod"
-											placeholder="Select"
+											:options="[CiemssMethodOptions.dopri5, CiemssMethodOptions.euler]"
+											@update:model-value="updateState"
 										/>
-									</td>
-								</tr>
-							</tbody>
-						</table>
-					</AccordionTab>
-				</Accordion>
-			</tera-drilldown-section>
+									</div>
+									<div class="label-and-input">
+										<label for="num-steps">Solver step size</label>
+										<tera-input-number v-model="knobs.extra.stepSize" />
+									</div>
+								</div>
+								<div class="spacer m-3" />
+								<h6 class="mb-2">Inference Options</h6>
+								<div class="input-row">
+									<div class="label-and-input">
+										<label for="num-iterations">Number of solver iterations</label>
+										<tera-input-number v-model="knobs.extra.numIterations" @update:model-value="updateState" />
+									</div>
+									<div class="label-and-input">
+										<label for="learning-rate">Learning rate</label>
+										<tera-input-number v-model="knobs.extra.learningRate" @update:model-value="updateState" />
+									</div>
+									<div class="label-and-input">
+										<label>Inference algorithm</label>
+										<tera-input-text disabled model-value="SVI" />
+									</div>
+									<div class="label-and-input">
+										<label>Loss function</label>
+										<tera-input-text disabled model-value="ELBO" />
+									</div>
+									<div class="label-and-input">
+										<label>Optimizer method</label>
+										<tera-input-text disabled model-value="ADAM" />
+									</div>
+								</div>
+							</fieldset>
+						</AccordionTab>
+					</Accordion>
+				</template>
+			</tera-slider-panel>
 		</section>
 		<section :tabName="CalibrateEnsembleTabs.Notebook">
```

Comment:
If you are going to change over to `DrilldownTabs` you can use it here as well

Reply:
Done!